src/styles.scss

```diff
@@ -19,3 +19,26 @@ div.govuk-prose-scope, p {
 td.govuk-table__cell {
   white-space: pre-wrap;
 }
+
```

Comment:
it would be good to grab this style from @hmcts/frontend if at all possible - speak to adam